paper-slider.html

```diff
@@ -90,6 +90,11 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) #sliderContainer {
```

Comment:
this generally works but we might want to just make `__isRTL` a property and use `reflectToAttribute: true` and query by `:host([__is-r-t-l])` or something to support nested `dir=` attributes (which I actually just got an email about)